cinema/views.py

```diff
@@ -43,6 +58,22 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = Movie.objects.all()
+        actors = self.request.query_params.get("actors")
+        genres = self.request.query_params.get("genres")
+        title = self.request.query_params.get("title")
+
+        if actors:
+            actors = actors.split(",")
+            queryset = queryset.filter(actors__id__in=actors)
```

Comment:
this works because Django tries to cover us, but the params type should be int here (or list of ints in this case)